pages/app/get-involved-detail/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint max-len: 0 */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import Page from 'layout/page';
+
+// Redux
+import withRedux from 'next-redux-wrapper';
+import { initStore } from 'store';
+// import { getInsights } from 'redactions/insights';
+
+import * as actions from 'layout/get-involved-detail/get-involved-detail-actions';
+
+import GetInvolvedDetail from 'layout/get-involved-detail';
+
+class GetInvolvedDetailPage extends Page {
+  static propTypes = {
+    user: PropTypes.object,
+    locale: PropTypes.string
+  };
+
+  static async getInitialProps(context) {
+    const props = await super.getInitialProps(context);
+
+    const { routes } = context.store.getState();
+
+    // Get static data
+    await context.store.dispatch(actions.fetchStaticData(routes.query.id));
+
+    // if (routes.query.id === 'submit-an-insight') {
```

Comment:
Are we going to get the insights or this is not needed anymore?

Reply:
The insights are currently static. I added to `get-involved-detail-default-state.js` with a comment to add actions once working.